### PR TITLE
[exporter/datadogexporter] Increase default `timeout` value

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** 

Increase default `timeout` value.

**Link to tracking Issue:** Relates to #2353

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Please add an entry in the changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## 💡 Enhancements 💡
+
+- `datadogexporter` increase default timeout to 30 seconds (#6131)
```
Comment (codeboten) [inline CHANGELOG.md:7]: The change is updating the timeout to 15 seconds, not 30 correct?

Comment (mx-psi) [reply CHANGELOG.md]: 🤦 sorry about that, should be fixed now

**Review by codeboten — APPROVED** [review]

**Comment by codeboten** [comment]
Please resolve the conflicts

**Review by MovieStoreGuy — APPROVED** [review]

**Pull request merged**